[AD-502] Fix bug with opening empty database.

Haskell-mouse opened the pull request:
Previously getState always tried to get the previous event
that happened before the given number of events.
This commit adds the handler for an empty database case.
Now `getState` will return special message instead of trying to get a previous event(end getting an error).

Commit:
[AD-502]Fix bug with opening empty database.

Previously getState always tried to get the previous event
that happened before the given number of events.
This commit adds the handler for an empty database case.
Now `getState` will return special message instead of trying to get a previous event(end getting an error).

artemohanjanyan commented on src/Data/Acid/Local.hs:
```diff
@@ -25,28 +26,27 @@ module Data.Acid.Local
+import System.Directory (createDirectoryIfMissing)
 import System.FileLock
-import System.Directory               ( createDirectoryIfMissing )
+import System.FilePath (takeDirectory, (</>))
```
Please don't change the formatting of the imports.

Haskell-mouse replied:
Ok, reverted it.

artemohanjanyan reviewed (commented):
Please put space after `[AD-502]` in a commit message

Commit:
[AD-502] Gets back imports formatting

Problem: In the previous commit the imports formatting
 was changed in Ariadne-style. It would be better to save original formatting.

Solution: Revert the changes.

epicallan reviewed (approved):
LGTM
